hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_unr_excl.el

```diff
@@ -0,0 +1,522 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: chencindy
+// Format Version: 2
+// Date: Thu May 13 14:31:19 2021
+// ExclMode: default
+//==================================================
+CHECKSUM: "2423918326 3905331873"
```

Comment:
@cindychip 
@msfschaffner 
It is worth setting up a meeting and going over these tool-generated exclusions using Verdi GUI to understand why there are so many items that are unreachable.

Reply:
Hey Sri, thanks so much for reviewing. Yes Michael and I did that this morning. We spend an hour going through every exclusion. 
The reason we have so many exclusions were because in otp_ctrl we try to categorize partitions into two big categories: part_buf and part_unbuf. But each partitions has its own unique features (HW partitions does need to de scramble/descramble data, LC partition does not have digest etc).
Thus we have many branches that are unreachable for certain partition.

@msf please feel free to add your input :)

Reply:
Actually something worth sharing - we did find one exclusion that currently tied to unreachable due to configurations, but Michael is considering enable that in the future. So even though this list is very tedious and long, but still worth going through for sure!

Reply:
Indeed, we did look at these exclusions and they are all OK.
As @cindychip mentioned, this is due to the fact that we reuse two parameterized partition submodules multiple times. The partition FSM branches / conditions behave differently, based on the OTP partition configuration. This leads to several branches that are unreachable in one partition instance, but reachable in another instance that is differently configured (for example scrambled and un-scrambled partitions).

Reply:
To @cindychip's second comment, we may have to repeat this exercise and look at the diff after changing the size of the `HW_CFG` partition (once we have connected all HW configuration bits in the design).